spec/models/vm_scan_spec.rb

```diff
@@ -1,13 +1,12 @@
 describe VmScan do
   context "A single VM Scan Job," do
     before(:each) do
-      @server = EvmSpecHelper.local_miq_server
+      @server = EvmSpecHelper.local_miq_server(:capabilities => {:vixDisk => true})
+      assign_smartproxy_role_to_server(@server)
 
       # TODO: We should be able to set values so we don't need to stub behavior
-      allow_any_instance_of(MiqServer).to receive_messages(:is_a_proxy? => true, :has_active_role? => true, :is_vix_disk? => true)
```

Comment:
implementation of `MiqServer#is_vix_disk?` just checking if current instance of `MiqServer` has `:SmartProxy` role and `MiqServer#capabilities[:vixDisk] == true'. 

We do not need to stub this behavior since needed role and capabilities assigned explicitly to instance of server on lines 4 and 5:

`      @server = EvmSpecHelper.local_miq_server(:capabilities => {:vixDisk => true})
      assign_smartproxy_role_to_server(@server)
`